service-endpoint/pyproject.toml

```diff
@@ -0,0 +1,44 @@
+[tool.setuptools]
+py-modules = []
+
+[project]
+name = "mittaridataputki-endpoint"
+description = ""
+readme = "README.md"
+requires-python = ">=3.9"
+dynamic = ["version"]
+
+dependencies = [
+  # "aiohttp",
+  "aiokafka",
+  # "aioredis",
+  "fastapi",
+  "fvhiot@https://github.com/ForumViriumHelsinki/FVHIoT-python/archive/refs/heads/master.zip",
+  "kafka-python",
+  # "msgpack",
```

Comment:
Is this functional without msgpack? data_pack() function uses it to serialise http request data. Also aioredis, aiohttp and xmltodict should be removed, if they are not needed @sheenacodes .

Reply:
rremoved commented line.
The service is functional as per current tests without msgpack